config/root/clusterworkspacetype-organization.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: tenancy.kcp.dev/v1alpha1
 kind: ClusterWorkspaceType
 metadata:
-  name: organization
+name: organization
 spec:
-  initializers:
-  - tenancy.kcp.dev/organization
+initializers:
+- tenancy.kcp.dev/organization
+defaultSubWorkspaceType: Team
+allowedSubWorkspaceTypes:
+- Team
+- Universal
```

Comment:
is `allowedParentTypes` coming later? `allowedSubWorkspaceTypes` does not really make sense IMO. Both Org and Team should rather define their parent types.